config/config.msft.yaml

```diff
@@ -30,6 +30,12 @@ defaults:
     etcd:
       kvName: arohcp-etcd-{{ .ctx.regionShort }}
       kvSoftDelete: true
+    istio:
+      istioctlUrl: "https://istio.io/downloadIstio"
+      istioctlVersion: "1.23.1"
+      tag: "prod-stable"
+      currentVersion: "asm-1-22"
```

Comment:
duplicate with line 16

Reply:
Gerd said he will use this version in the future.

Reply:
Can't this PR just do that? Just need to change the templates to use this one.